### PR TITLE
add lightning example to frontpage

### PR DESCRIPTION
**Pull request by mathisonian**
Per @Carreau's [suggestion](https://github.com/lightning-viz/lightning-viz.github.io/pull/1), this pull request adds a link to the [Lightning Projects](https://github.com/lightning-viz/lightning)'s example notebooks. I'm happy to make any updates that you think appropriate.

Thanks!
Matt

example image

### CODE REVIEW

File: nbviewer/frontpage.json [hunk 1]
```diff
@@ -81,11 +86,6 @@
         "text": "Jaynes-Cummings model",
         "target": "github/jrjohansson/qutip-lectures/blob/master/Lecture-1-Jaynes-Cumming-model.ipynb",
         "img": "/static/img/example-nb/jaynes-cummings.png"
-      },
-      {
-        "text": "SymPy notebook",
-        "target": "github/ipython/ipython/blob/2.x/examples/Notebook/SymPy.ipynb",
-        "img": "/static/img/example-nb/sympy.png"
```
Comment (rgbkrk) [inline nbviewer/frontpage.json]: This one probably didn't need to be cut as part of this PR, but this is referencing a soon to be old branch.

**Comment by rgbkrk** [comment]
Looking great, not sure what made the build stall out on Travis.

**Comment by mathisonian** [comment]
Thanks Kyle,

I just removed the SymPy example assuming these were going in chronological order, but will update the PR with whatever arrangement makes sense

**Comment by rgbkrk** [comment]
Ah, no it's ok. I think that one is likely stale compared to more recent SymPy content. It's ok as I'd rather see a multiple of 3 notebooks on the front page. x mod 3 in 3-x mod 3 out.

**Comment by mathisonian** [comment]
Hey, just wanted to bump this. Is there anything I need to do to get the Travis build to pass?

**Comment by rgbkrk** [comment]
I just re-triggered the build. I had assumed that Travis does a merge commit. Perhaps something was different in the `.travis.yml` from when you forked and what our current setup expects.

I'll go ahead and merge this as the JSON is fine.

File: nbviewer/frontpage.json [hunk 2]
```diff
@@ -43,6 +43,11 @@
     "header":"Misc",
     "links":[
       {
+        "text": "Data Visualization with Lightning",
+        "target": "github/lightning-viz/lightning-example-notebooks/blob/master/index.ipynb",
+        "img": "/static/img/example-nb/lightning.png"
+      }
```
Comment (rgbkrk) [inline nbviewer/frontpage.json]: Oops, missing a comma here. @bollwyvl fixed it in a later PR.

**Pull request merged**